packages/remix-dev/__tests__/cli-test.ts

```diff
@@ -23,36 +24,86 @@ describe("remix cli", () => {
       let { stdout } = await execFile("node", [remix, "--help"]);
```

Comment:
Let's run it with `--no-color` so we don't get the ANSI escape sequences in the output.

Reply:
😬 [`a76fd5b` (#2189)](https://github.com/remix-run/remix/pull/2189/commits/a76fd5b78dacdcda3f0f4a0949356915206405f3)

I tried --no-colors, but it threw a fit, let me try again

Reply:
🤨 `Command failed: node /Users/loganmcansh/Developer/github.com/remix-run/remix/build/node_modules/@remix-run/dev/cli.js --help --no-colors`